tests/kernel/pipe/pipe_api/src/test_pipe_contexts.c

```diff
@@ -142,7 +142,9 @@ static void tpipe_put_small_size(struct k_pipe *ppipe, k_timeout_t timeout)
 	for (int i = 0; i < PIPE_LEN; i += wt_byte) {
 		/**TESTPOINT: pipe put*/
 		to_wt = 15;
-		k_pipe_put(ppipe, &data[i], to_wt, &wt_byte, 1, timeout);
+		zassert_equal(k_pipe_put(ppipe, &data[i], to_wt, &wt_byte, 1, timeout),
+			0, "put some data failed");
```

Comment:
I suggest just using zassert_false and using NULL as the message so that it's consistent with tpipe_put/tpipe_get/tpipe_get_small_size.

Reply:
Hi @s-kelvin , updated, could you take a look at again, thanks.